src/components/forms/PlanForm/index.tsx

```diff
@@ -986,18 +988,36 @@ const PlanForm = (props: PlanFormProps) => {
                               />
                             </FormGroup>
                           </fieldset>
-                          {actionTriggers.hasOwnProperty(values.activities[index].actionCode) && (
-                            <fieldset className="triggers-fieldset">
-                              <legend>{TRIGGERS_LABEL}</legend>
-                              {actionTriggers[values.activities[index].actionCode]}
-                            </fieldset>
-                          )}
-                          {actionConditions.hasOwnProperty(values.activities[index].actionCode) && (
-                            <fieldset className="conditions-fieldset">
-                              <legend>{CONDITIONS_LABEL}</legend>
-                              {actionConditions[values.activities[index].actionCode]}
-                            </fieldset>
-                          )}
+                          <Accordion
```

Comment:
Could we have used [Collapse from reactstrap](https://reactstrap.github.io/components/collapse/) instead of doing it this way?

Reply:
yeah sure, I was looking for Accordion , did not find it.